jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ChunkAccumulator.java

```diff
@@ -222,6 +223,16 @@ public void run()
             }
         }
 
+        @Override
+        public InvocationType getInvocationType()
+        {
+            return InvocationType.NON_BLOCKING;
```

Comment:
That's not correct, as it may invoke user code.

Reply:
This is another case when we need to know how it is to be used.  If only get is called, then it is NON_BLOCKING.